UnitySetup/UnitySetup.psm1

```diff
@@ -964,6 +964,7 @@ function Start-UnityEditor {
             if ( $Wait ) {
                 if ( $process.ExitCode -ne 0 ) {
                     if ( $LogFile -and (Test-Path $LogFile -Type Leaf) ) {
+                        Write-UnityErrors $LogFile
```

Comment:
Running Unity with `-BatchMode -Quit -Wait` but without executing a command will exit with 0 exit code, though the logs contain the compilation errors. We should pull this error detection out of the dependency on the exit code.